eclair-core/src/main/scala/fr/acinq/eclair/payment/receive/MultiPartHandler.scala

```diff
@@ -395,7 +426,8 @@ object MultiPartHandler {
                     case Some(payment: IncomingStandardPayment) => replyTo ! ProcessPacket(packet.add, payload, Some(payment))
                     case None => replyTo ! ProcessPacket(packet.add, payload, None)
                   }
-                case payload: FinalPayload.Blinded =>
+                  Behaviors.stopped
+                case payload: FinalPayload.Blinded if payload.pathId.length == 32 =>
```

Comment:
This isn't very robust, can't we instead look for tlvs in the payload that are only added for DB-less invoices? That could be exposed as a boolean field directly defined in `FinalPayload.Blinded`.

Reply:
Blinded payments are now always DB-less.